__tests__/commands/install/bin-links.js

```diff
@@ -219,7 +219,7 @@ describe('with focus', () => {
           true,
         );
         expect(await linkAt(config, 'node_modules', '.bin', 'example-yarn-workspace-2')).toEqual(
-          '../example-yarn-workspace-2/index.js',
+          '../../../example-yarn-workspace-2/index.js',
```

Comment:
Interesting .. I've simply updated this test, since the new value seems more correct to me. The extra `../..` is required to jump out of `node_modules/.bin` (the `linkAt` function just reads the symlink as it is, it doesn't make it relative to anything special).